src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/InputHelper.java

```diff
@@ -30,4 +46,105 @@ public static InputScannedDoc getSampleInputDocument(String suffix) {
     public static InputScannedDoc getSampleInputDocument() {
         return getSampleInputDocument("");
     }
+
```

Comment:
This helper defines a lot of values. I think this should be tests' job, so that it's clear where the values come from.

Reply:
At the moment actual values do not matter. We can even generate random values in this class. What matters in the test is that the values are properly transferred from one object to another.

Reply:
I agree, but currently those values are distributed (not only across methods, but also classes), and it's very hard to track where those values come from any why we're expecting them and not any other values. Ideally, a test should say:
- this is the input data
- call the tested method
- this is the expected result/passed data/...
- assert that actual is as expected
We should literally see it just looking at the code of the test method. For this to happen, we need to pass values to helper methods, as those methods shouldn't be bound to any particular test (and contain an invisible dependency).